Stp.Data/Entities/TestSectionAndTask.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Stp.Data.Entities
+{
```

Comment:
[Table(nameof(TestSectionAndTask))] attribute is needed. Without this attribute, the table name generated according to the name of the corresponding DbContext field (TestSectionAndTaskS)  - which is incorrect.